src/lib.rs

```diff
@@ -818,27 +815,24 @@ fn graph_floyd_warshall_numpy(
 ///     path between two nodes then the corresponding matrix entry will be
 ///     ``np.inf``.
 /// :rtype: numpy.ndarray
-#[pyfunction(as_undirected = "false")]
-#[text_signature = "(graph, weight_fn, /, as_undirected=False)"]
+#[pyfunction(as_undirected = "false", default_weight = "1.0")]
+#[text_signature = "(graph, weight_fn, /, as_undirected=False, default_weight=1.0)"]
```

Comment:
#[text_signature = "(graph, /, weight_fn=None, as_undirected=False, default_weight=1.0)"]
as well as the others?

Reply:
Good catch, yeah I missed that. Updated in https://github.com/Qiskit/retworkx/pull/158/commits/f2d3b7792e8fc925920c5d0a62772a4f05a173e9